model.py

```diff
@@ -20,12 +20,26 @@ class AliasMatch(orm.Document):
 
 
 class Match(orm.Document):
-    fields = [('winner', orm.ObjectIDField(required=True)),
-              ('loser', orm.ObjectIDField(required=True))]
+    fields = [('match_id', orm.IntField(required=True)),
+              ('winner', orm.ObjectIDField(required=True)),
+              ('loser', orm.ObjectIDField(required=True)),
+              ('excluded', orm.BooleanField(required=True))]
 
     def __str__(self):
         return "%s > %s" % (self.winner, self.loser)
 
+    def get_match_id(self):
+        return self.match_id
+
+    def set_match_id(self, id):
+        self.match_id = id
+
+    def is_excluded(self):
+        return self.excluded
+
+    def set_excluded(self, excluded):
+        self.excluded = excluded
+
```

Comment:
So this one was actually because something wasn't working right and I was trying something out. I don't believe I ever actually use this, but thought it could have been something to do with the file I was working in.